src/miniscript/mod.rs

```diff
@@ -156,6 +157,20 @@ impl<Ctx: ScriptContext> Miniscript<Ctx::Key, Ctx> {
     /// insane scripts. In general, in a multi-party setting users should only
     /// accept sane scripts.
     pub fn parse_insane(script: &script::Script) -> Result<Miniscript<Ctx::Key, Ctx>, Error> {
```

Comment:
In 89169c23aec0bac4009593e6c451bf9b0432ee6d:

I think we should deprecate this and tell users to use `parse_with_ext` with `ExtParams::all` (or whatever we call that). What do you think?

Reply:
There is a small difference between `insane` and `ext` (experimental stuff). Insane things are well documented and specified in spec, but disallowed for user safety. The `raw_pkh` stuff is not specified anywhere yet.

Reply:
I think having an insane high level API is still valuable short-hand for `parse_with_ext(ExtParams::insane())` which is different from `parse_with_ext(ExtParams::all())`

Reply:
Ok, cool, that's fair. I didn't realize that "insane" covered well-defined things outside of this library.